.gitignore

```diff
@@ -12,6 +12,8 @@ dist
 dist-ssr
 *.local
 
+*~
```

Comment:
Do we need this change?

Reply:
Right, Vim. LGTM

Reply:
Or, In my case, Emacs. (But same reason anyway)